### PR TITLE
Populate property distance data

### PR DESCRIPTION
**Pull request by ChrisPalmerAND**

### CODE REVIEW

File: packages/backend/services/propertyService.js [hunk 1]
```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
...
   return [...new Set(propertiesWithinPolygons.flat())];
 };
 
-const getDistance = async (startCoordinates, endCoordinates, methodOfTravel) => {
+const getDirections = async (
+  startCoordinates,
+  endCoordinates,
+  methodOfTravel
+) => {
   const token = `?access_token=${process.env.MAPBOX_TOKEN}`;
```
Comment (Aloruss) [inline packages/backend/services/propertyService.js:59]: i would have only the token here and add ?access_token= to the url below

Comment (Aloruss) [inline packages/backend/services/propertyService.js]: i would have only the token here and add ?access_token= to the url below
```suggestion
  const token = `${process.env.MAPBOX_TOKEN}`;
```

File: packages/backend/services/propertyService.js [hunk 2]
```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
...
-
   const { data } = await axios.get(
-    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates}${endCoordinates}?access_token=${token}`
+    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates};${endCoordinates}${token}`
```
Comment (Aloruss) [inline packages/backend/services/propertyService.js]: is the ; required between start and end coordinates?
```suggestion
    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates};${endCoordinates}?access_token=${token}`
```

Comment (ChrisPalmerAND) [reply packages/backend/services/propertyService.js]: yes that's how they separate the start and end from each other. then the long and lat are separated with a comma. Go home it's late

File: packages/backend/services/propertyService.js [hunk 3]
```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
...
-  return resp.data;
+const calculateTravelToWork = async (property) => {
+  const endCoordinates = AND_DIGITAL_COORDINATES;
+
+  const startCoordinates = [
+    property.details.coordinates[1],
+    property.details.coordinates[0],
+  ];
+
+  for (let i = 0; i < METHOD_OF_TRAVEL.length; i++) {
```
Comment (Aloruss) [inline packages/backend/services/propertyService.js]: a for of should work aswell here

File: packages/backend/services/propertyService.js (hunk 1 above)
Comment (ChrisPalmerAND) [reply packages/backend/services/propertyService.js]: yeh that's cool!

File: packages/backend/services/propertyService.js (hunk 3 above)
Comment (ChrisPalmerAND) [reply packages/backend/services/propertyService.js]: What are the advantages? Is it worth changing or just letting me know that there are other options?

Comment (ChrisPalmerAND) [reply packages/backend/services/propertyService.js]: Looked into this and like how clean it is so just swapping it over

File: packages/backend/services/propertyService.js [hunk 4]
```diff
@@ -1,61 +1,82 @@
...
-const getNearestWhat = async (pointOfInterest, coordinates) => {
-  const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${pointOfInterest}.json?type=poi&proximity=${coordinates}&access_token=${token}`;
-  const resp = await axios.get(url);
-  return resp.data;
+const calculateTravelToWork = async (property) => {
+    const endCoordinates = AND_DIGITAL_COORDINATES;
+
+    const startCoordinates = [property.details.coordinates[1], property.details.coordinates[0]];
+
+    for (const i of METHOD_OF_TRAVEL) {
```
Comment (Aloruss) [inline packages/backend/services/propertyService.js]: pickiness... const methodOfTravel of METHODS_OF_TRAVEL would be nicer. up to you, though

**Review by Aloruss — APPROVED** [review]

**Pull request merged**